_CS141/laws.md

```diff
@@ -56,3 +56,28 @@ m >>= return = m
 | `f x * g y` | *f(x) &times; g(y)* |
 
 {:.centeredtable}
+
+> Function **expressions** associates to the right.
```

Comment:
I am not really sure if it makes sense to include associativities under "laws" since they are more about the syntax of the language (while laws are about the behaviour of functions), but this is a very minor niggle of course!